recipes/r-rcatch22/meta.yaml

```
@@ -0,0 +1,95 @@
+{% set version = '0.2.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rcatch22
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/Rcatch22_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/Rcatch22/Rcatch22_{{ version }}.tar.gz
+  sha256: b4e173ba89da2339a949a649f9b3358f0cf95c744ea9d386a9883a846c57f9bd
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
```

This will address the Win error. (see https://github.com/conda/conda-build/pull/4786)

```suggestion
  missing_dso_whitelist:
    - '*/R.dll'            # [win]

```